jerry-core/ecma/operations/ecma-promise-object.c

```diff
@@ -437,6 +467,25 @@ ecma_promise_create_resolving_functions (ecma_object_t *object_p, /**< the promi
   /* 5. - 7. */
   funcs->reject = ecma_promise_create_resolving_functions_helper (object_p,
                                                                   ecma_promise_reject_handler);
+  if (!create_already_resolved)
+  {
+    return;
+  }
+
+  ecma_value_t already_resolved = ecma_op_create_boolean_object (ECMA_VALUE_FALSE);
+  ecma_string_t *str_already_resolved_p = ecma_get_magic_string (LIT_INTERNAL_MAGIC_STRING_ALREADY_RESOLVED);
+
+  ecma_op_object_put (ecma_get_object_from_value (funcs->resolve),
```

Comment:
I'd rather use create_namedata_property here since, since both `funcs->resolve` and `funcs->reject` created above should not have existing `LIT_INTERNAL_MAGIC_STRING_ALREADY_RESOLVED` property.